<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="inputMiasto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>331</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPobierzStacje">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Pobierz stacje</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboStacje">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textWyniki">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>110</y>
      <width>371</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboSensory">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz stacje</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz miasto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wybierz czujnik</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPobierzDane">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Pobierz dane</string>
    </property>
   </widget>
   <widget class="QFrame" name="chartView">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>641</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonOffline">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Offline</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboZakres">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateOd">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateDo">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>110</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabelOd">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Data od</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelDo">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>190</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Data do</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
